examples/hello-monai/config/bundle_configs/train.json

```diff
@@ -0,0 +1,303 @@
+{
```

Comment:
Please don't put the hard-code config of a bundle here.
Let's tell users how to download the spleen bundle and use the config file of downloaded bundle directly.

Thanks.